### PR TITLE
M2131 solution updates

### PR DESCRIPTION
**Pull request by thbanasi**
## Before submitting this PR please ensure that you have read the following sections and then completed the template below:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures, there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

## After the submission has been made, please look at the Validation Checks:

> Check that the validations are passing and address any issues that are present. Let us know if you have tried fixing and need help.

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

## PR Template

-----------------------------------------------------------------------------------------------------------
 **Description for the PR:**
  (Enter the description below)

**Testing Completed:**
  Yes/ No : 

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json [hunk 1]
```diff
@@ -1174,7 +1209,7 @@
           {
             "type": 1,
             "content": {
-              "json": "### Pricing\r\n✳️[Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️[Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"
+              "json": "### Pricing\r\n✳️ [Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️ [Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"
```
Comment (shainw) [inline Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json:1212]: I noticed you removed some of the icons in other parts, did you want to remove here also or not?

**Review by shainw — APPROVED** [review]
Assuming you left some of the icons in the Workbook, should be good here.

**Comment by thbanasi** [comment]
@Shain Wray ***@***.***>,

Thanks so much for the update and for your support. Icons don’t display in color anymore in panel titles so I removed them from the panel titles in the Assessment section. In the Log Management Section I added a space after the icons for visual effect, but agreed I do want to keep them there.

Very Respectfully,

TJ Banasik
CISSP-ISSEP,ISSAP,ISSMP
Senior Program Manager
C&AI, Sovereign Cloud Security Group
***@***.******@***.***>
(202) 658-8148



***@***.***

LinkedIn: In/tjbanasik<http://www.linkedin.com/in/tjbanasik>

File: Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json (hunk 1 above)
Comment (shainw) [reply Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json:1212]: Chatted with TJ - this is by design.  Closing ocmment.

**Pull request merged**